### PR TITLE
Adding tips and pagination.max tips to Element Builder guide

### LINKED ISSUE #953: Elements Builder needs a tips section
## Description
Element Builder needs a tips section and it needs it like a 🚲  needs a rider and a 🐟  needs 💧

### LINKED ISSUE #954: Pagination Max can now be an object
## Description
See title 👆 . Now, what does that mean? What was once a integer field can now be a fun map!

### PR DESCRIPTION
**Pull request by tyler-cloud-elements**
## Highlights
* See title 👆 

## Closes
* Closes #953 
* Closes #954

### CODE REVIEW

File: docs/guides/element-builder/tips.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 50
+sitemap: false
+---
+# Tips
+
+Tips is a section to find all of your tips and tricks for building high quality elements! *More to come!*
+
+Have a tech tip suggestion? Please contact [Support](mailto:support@cloud-elements.com).
```
Comment (davewilks) [inline docs/guides/element-builder/tips.md]: Have them contact the Documentation Team at documentation@cloud-elements.com.

File: docs/guides/element-builder/tips.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+### **Pagination Max**
```
Comment (davewilks) [inline docs/guides/element-builder/tips.md]: Start with a Heading two: ## and no need to make bold.

File: docs/guides/element-builder/tips.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+When building an element in Element Builder, you set up a max pagination, (*number 10 pictured below*) for your element. Cloud Elements uses this to discern what to send to the vendor as a default in the __pageSize__ parameter when making GET list request.
```
Comment (davewilks) [inline docs/guides/element-builder/tips.md]: Make sure that we're consistently referring to the same thing: "Pagination Max", "max pagination", "pageSize" and "Max Page Size" (in the UI) are all referring to the same thing. Can we go with what is in the UI and the name of the parameter? So the Heading would be Max Page Size. The first sentence would be "..., you set up Max Page Size"

Also, regarding the numbered image. I think it'd be a lot clearer if only Max Page Size had a rectangle around it. The numbers are overwhelming.

And a couple of small edits: "...:pencil2:~~__pageSize__~~ `pageSize` parameter when making :pencil2:a (add "a" :point_left: **or** make it plural: "add 'GET list` requests")* `GET list` (:question:should this be `GET /list`?) request.

File: docs/guides/element-builder/tips.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+![Element Builder Configuration 1](http://cloud-elements.com/wp-content/uploads/2015/04/Configuration1.png)
+
+
+You will notice that __Twitter__ uses __100__ as it's max page size (or limit). This means that whenever making GET list requests in __Twitter__, Cloud Elements will use __100__ as it's page size whenever it hasn't been supplied by the end user. However, there may be times where the endpoint may have different max page sizes for a resource. For example, when you are calling a `GET /users` in [FreshService](http://api.freshservice.com/#view_all_user), they note that `users` only returns `50` results per page. However, when calling `GET /problems` in [FreshService](http://api.freshservice.com/#view_all_problem), they note that `problems` only returns `30` results per page. To solve this, you can pass an object that specifies the pagination for each object, as well as the default.
```
Comment (davewilks) [inline docs/guides/element-builder/tips.md]: ":pencil2:~~You will notice that __Twitter__ uses __100__ as it's max page size (or limit).~~ The image above shows that Twitter uses `100` as the **Max Page Size**. "
"This means that whenever making ~~GET list~~ :pencil2:`GET /list` requests in ~~__Twitter__~~:pencil2:Twitter, Cloud Elements ~~will use~~ :pencil2:uses ~~__100__~~:pencil2:100 as ~~it's~~:pencil2:the :pencil2:default page size ~~whenever it hasn't been supplied by the end user~~. "

File: docs/guides/element-builder/tips.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+![Element Builder Configuration 2](/assets/img/element-builder/tips/freshservice-configuration-pageMax.png)
+
+#### Examples
```
Comment (davewilks) [inline docs/guides/element-builder/tips.md]: Let's make this ### instead of ####

File: docs/guides/element-builder/tips.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+1. Setting pagination max for the `users` and `problems` resources:
```
Comment (davewilks) [inline docs/guides/element-builder/tips.md]: The one thing I think is missing here is context. Are these parameters inside a pageSize object? Like should this be:
"pageSize":{
"users": 50,
...
}

Comment (tyler-cloud-elements) [inline docs/guides/element-builder/tips.md]: No, in this case - it's just a flat json object

File: docs/guides/element-builder/tips.md (hunk 3 above)
Comment (tyler-cloud-elements) [inline docs/guides/element-builder/tips.md]: Makes sense 👍 

I was just borrowing the Twitter image from the 'Build an Element from UI' page, but I can add a different one. 

No, it's not a `GET /lists` request per se. It's a GET request that returns a list response (_which could be a `GET /lists` request, or `contacts`, `users`, etc._). I couldn't really think of the best phrasing for that. What would you recommend there?

**Pull request merged**